chapters/object_types/spatial_fields.txt

```diff
@@ -55,6 +55,39 @@ their addresses in memory can be easily computed from a 3D position. A
 common type of structured spatial fields are regular grids.
 ====
 
+[[object_types_spatial_field_nanovdb]]
+==== NanoVDB
+
+Extension `KHR_SPATIAL_FIELD_NANOVDB`
+
+NanoVDB spatial fields are created by passing the subtype string
+`nanovdb` to <<object_types_spatial_field, `anariNewSpatialField`>>.
+The parameters understood by NanoVDB spatial fields are summarized in
+the table below.
+
+.Configuration parameters for NanoVDB spatial fields.
+[cols="<,<2,>,<5",options="header,unbreakable"]
+|===================================================================================================
+| Name    | Type                |    Default | Description
+| data    |`ARRAY1D` of `UINT8` |            | NanoVDB grid, as a binary blob.
+| filter  |`STRING`             |    `linear`| filter used for reconstructing the field,
+                                               possible values: `nearest`, `linear`, `cubic`
+|===================================================================================================
+
+The `data` parameter content is expected to be a NanoVDB grid, as read from a NanoVDB file or constructed by the NanoVDB API.
+The spatial field extent is implicitely defined in the grid header.
+
+Setting `data` to an incompatible or invalid NanoVDB grid should result in the device implementation emitting a warning and ignoring the invalid object.
+
+[NOTE]
+.Note
+====
+Incompatible NanoVDB grids are device dependent and could result from an unsupported file version or unsupported data type (such as quantized fp4 or not so common data types in rendering workflows).
```

Comment:
So, an implementation may support a CPU and GPU device, both supporting NanoVDB, but potentially not the same data types? Or even the same GPU device supports different data type depending on the compute capabilities of the installed GPU? Any way to add a device property to query the supported data types then?